classes/observer.php

```diff
@@ -186,7 +186,7 @@ public static function email_mention_hsu(\mod_hsuforum\event\assessable_uploaded
                         $body = get_config('local_mention_users', 'defaultproperties_body');
                         $body = str_replace("{student_first_name}", $taggedusername, $body);
                         $body = str_replace("{coach_first_name}", $course_coach->firstname, $body);
-                        $body = str_replace("{post_link}", $link, $body);
+                        $body = str_replace("{post_link}", 'http://' .$link, $body);
```

Comment:
Super minor nit but missing a space between . and $link

Reply:
thanks, fixed this